x-pack/solutions/security/plugins/kubernetes_security/public/components/tree_view_container/helpers.ts

```diff
@@ -30,6 +30,7 @@ export const KUBERNETES_COLLECTION_FIELDS: KubernetesCollectionMap = {
   containerImage: CONTAINER_IMAGE_NAME,
 };
 
+// TODO: update color scheme for Borealis
```

Comment:
k8s_security should be decommissioned in 9.0 and serverless, therefore not touching, but leaving a comment in case the dashboard will be revived in the future